src/Arblib.jl

```diff
@@ -41,6 +42,25 @@ macro libflint(function_name)
     return (:($function_name), Arb_jll.libflint)
 end
 
+const __isthreaded = Ref(false)
+
+function __init__()
+    __isthreaded[] = Base.get(ENV, "NEMO_THREADED", "") == "1"
+
+    if __isthreaded[]
+        ccall((:__gmp_set_memory_functions, :libgmp), Nothing,
+            (Int, Int, Int), 0, 0, 0)
+    end
+end
+
+function flint_set_num_threads(a::Int)
+    if !__isthreaded[]
+        error("To use threaded flint, julia has to be started with NEMO_THREADED=1")
+    else
+        ccall(@libflint(flint_set_num_threads), Nothing, (Int,), a)
```

Comment:
is that a `Clong`, or `Cint` in flint?

Reply:
It's actually `Cint`: https://flintlib.org/doc/flint.html?highlight=flint_set_num_threads#c.flint_set_num_threads
```suggestion
        ccall(@libflint(flint_set_num_threads), Nothing, (Cint,), a)
```